doc/PlayerBot/todo.txt

```diff
@@ -0,0 +1,40 @@
+ToDo
```

Comment:
This is supposed to be already done, this file is missing on my disk... Another weird git thingy.